src/road/traffic.py

```diff
@@ -66,6 +66,21 @@ def __init__(self, id, node: RoadSegmentNode):
         # Intersection
         self.wait_time = 0  # sec
 
+    @property
+    def speed(self):
+        """
+        Returns the speed of the vehicle in terms of tiles traveled
+        """
+        return self.vehicle_speed * tw
+    
+    @speed.setter
+    def speed(self, value):
+        """Sets the vehicle speed. Expects a value in terms of tile size"""
+        if isinstance(value, int):
+            self.vehicle_speed = value // tw # Floored for int
+        else:
+            raise TypeError("Value must me int not {}".format(type(value)))
```

Comment:
Since this project uses Python 3.7.4, let's use an f-string here instead of `format()`.

Reply:
As a bonus, if you're aware of good technologies for enforcing type hint checking in Python, you could add that as well. Then we can just specify `value: int` in the function header.